src/client/users.rs

```diff
@@ -122,6 +171,18 @@ pub struct User {
     pub created_at: OffsetDateTime,
 }
 
+/// Binds a [`User`] to a [`Tenant`] for a `frontegg.user.*` webhook event
+///
+/// [`Tenant`]: crate::client::tenant::Tenant
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct WebhookTenantBinding {
+    /// The ID of the tenant.
+    pub tenant_id: Uuid,
+    /// The roles to which the user belongs in this tenant.
+    pub roles: Option<Vec<Role>>,
```

Comment:
user.tenants[*].roles field can be missing on `frontegg.user.enrolledMFA` events

Reply:
Ditto.